Update version links and documentation to reflect v2.0-RC (#221)
MobilityData/gbfs: gbfs.md

`license_id` <br/>*(added in v2.0-RC)* | Conditionally required | String | Required if the dataset is provided under a standard license. An identifier for a standard license from the [SPDX License List](https://spdx.org/licenses/). Provide `license_id` rather than `license_url` if the license is included in the SPDX License List. See the GBFS wiki for a [comparison of a subset of standard licenses](data-licenses.md). If the license_id or license_url fields are blank or omitted, this indicates that the feed is provided under the [Creative Commons Universal Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/legalcode).
`license_url` | Conditionally required <br/>*(as of v2.0-RC)* | URL | Required if the dataset is  provided under a customized license. A fully qualified URL of a page that defines the license terms for the GBFS data for this system. Do not specify a `license_url` if `license_id` is specified. If the license_id or license_url fields are blank or omitted, this indicates that the feed is provided under the [Creative Commons Universal Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/legalcode). *(as of v2.0-RC)*

> `attribution_organization_name` <br/>*(added in v2.0-RC)* | Optional | If the feed license requires attribution, name of the organization to which attribution should be provided.

Reviewer: ```suggestion
`attribution_organization_name` <br/>*(added in v2.0-RC)* | Optional | String | If the feed license requires attribution, name of the organization to which attribution should be provided.
```

Author: Thanks! I fixed in https://github.com/NABSA/gbfs/pull/221/commits/8d7c56ef46905d7594783ea70e888b45f9f1a244.

> `attribution_url` <br/>*(added in v2.0-RC)* | Optional | URL of the organization to which attribution should be provided.

Reviewer: ```suggestion
`attribution_url` <br/>*(added in v2.0-RC)* | Optional | URL | URL of the organization to which attribution should be provided.
```

Author: Thanks! I fixed in https://github.com/NABSA/gbfs/pull/221/commits/8d7c56ef46905d7594783ea70e888b45f9f1a244.

`stations` | Yes | Array | Array that contains one object per station in the system as defined below.
\-&nbsp;`station_id` | Yes | ID | Identifier of a station see [station_information.json](#station_informationjson).
\-&nbsp;`num_bikes_available` | Yes | Non-negative integer | Number of bikes available for rental. Number of functional bikes physically at the station. To know if the bikes are available for rental, see `is_renting`.

> \-&nbsp;`num_bikes_disabled` | Optional | Non-negative integer | Number of disabled bikes at the station. Vendors who do not want to publicize the number of disabled bikes or docks in their system can opt to omit station capacity (in station_information), `num_bikes_disabled` and `num_docks_disabled` *(as of v2.0-RC)*. If station capacity is published then broken docks/bikes can be inferred (though not specifically whether the decreased capacity is a broken bike or dock).

Reviewer: ```suggestion
\-&nbsp;`num_bikes_disabled` | Optional | Non-negative integer | Number of disabled bikes at the station. Vendors who do not want to publicize the number of disabled bikes or docks in their system can opt to omit station capacity (in station_information), `num_bikes_disabled` and `num_docks_disabled` *(as of v2.0-RC)*. If station capacity is published, then broken docks/bikes can be inferred (though not specifically whether the decreased capacity is a broken bike or dock).
```

Field Name | Required | Type | Defines
--|--|--|--
`bikes` | Yes | Array | Array that contains one object per bike that is currently stopped as defined below.

> \-&nbsp;`bike_id` | Yes | ID | Identifier of a bike, rotated to a random string, at minimum, after each trip to protect privacy *(as of v2.0-RC)*. Note: Persistent bike_id, published publicly, could pose a threat to individual traveler privacy.

Reviewer: This table still seems to have a problem with wrapping.

Author: Yes, I saw that too. But I'm not sure how to fix it. It seems to be formatted the same as station_information.json, and that wraps...

\-&nbsp;`lat` | Yes | Latitude | Latitude of the bike.
\-&nbsp;`lon` | Yes | Longitude | Longitude of the bike.

> \-&nbsp;`is_reserved` | Yes | Boolean | Is the bike currently reserved? <br /><br /> `1` - Bike is currently reserved. <br /> `0` - Bike is not currently reserved.

Reviewer: ```suggestion
\-&nbsp;`is_reserved` | Yes | Boolean | Is the bike currently reserved? <br /><br /> `true` - Bike is currently reserved. <br /> `false` - Bike is not currently reserved.
```

> \-&nbsp;`is_disabled` | Yes | Boolean | Is the bike currently disabled (broken)? <br /><br /> `1` - Bike is currently disabled. <br /> `0` - Bike is not currently disabled.

Reviewer: ```suggestion
\-&nbsp;`is_disabled` | Yes | Boolean | Is the bike currently disabled (broken)? <br /><br /> `true` - Bike is currently disabled. <br /> `false` - Bike is not currently disabled.
```
